src/components/List.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react"
+import { fetchAllReviews } from "../api"
+export default function List (){
+  const [reviews, setReviews] = useState([])
+  useEffect(()=>{
+    fetchAllReviews().then((reviews)=>{
+      setReviews(reviews)
+    })
+  })
+
+  const displayReviews = (review)=>{
+    return (
```

Comment:
Need to add a loading state on this page - this will tell user it is loading if waiting for data to come through :)